languages/python/oso/tests/test_polar.py

```diff
@@ -894,5 +894,41 @@ def test_unexpected_expression(polar):
     """Ensure expression type raises error from core."""
     polar.load_str("f(x) if x > 2;")
 
-    with pytest.raises(UnexpectedPolarTypeError):
+    with pytest.raises(exceptions.UnexpectedPolarTypeError):
         next(polar.query_rule("f", Variable("x")))
+
+
+def test_lookup_in_head(polar):
```

Comment:
Do we need the library tests if we have the regression test in the core?

Reply:
I don't mind keeping it, but we might consider moving it to the parity test suite. What do you think?

Reply:
Why parity tests? Is this a feature we supported in the old Oso?

Reply:
Ah, there's an unfortunate nomenclature clash in our codebase.

You're talking about [the old Python parity tests](https://github.com/osohq/oso/tree/c453e478cf7dbde9ccb12d2ba73fca9915b8ca0d/languages/python/oso/tests/parity), which we've been meaning to remove from the codebase for a while. The only thing stopping us in the past was ensuring that we had captured *all* of the test cases in the old parity test suite in the new Python test suite. Those are effectively obsolete, and we should really clean them up one of these days.

I'm talking about [the language parity tests](https://github.com/osohq/oso/tree/c453e478cf7dbde9ccb12d2ba73fca9915b8ca0d/test), which exist to ensure parity of functionality across the various language integrations.

Reply:
Aha, yes makes sense. Okay I can move it to the language parity tests.